### PR TITLE
Update assets repo

### PR DESCRIPTION
**Pull request by kolya182**
Updating `Assets` due to recent changes

### CODE REVIEW

File: assets/add_new_asset.md [hunk 1]
```diff
@@ -139,20 +185,26 @@ https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/bitcoin/
...
 
+TRC-20:
+```js
+https://raw.githubusercontent.com/trustwallet/assets/master/blockchains/tron/assets/TR7NHqjeKQxGTCi8q8ZY4pL8otSzgjLj6t/logo.png
+```
+
 ## Used in Applications
 - [Trust Wallet](https://trustwallet.com) - [iOS](https://itunes.apple.com/us/app/trust-ethereum-wallet/id1288339409) and [Android](https://play.google.com/store/apps/details?id=com.wallet.crypto.trustapp)
-- [0x Tracker](https://0xtracker.com) - The 0x Protocol Trade Explorer and news aggregator.
-
```
Comment (vikmeup) [inline assets/add_new_asset.md]: no need to include this here, only keep `## Used in Applications` on github

File: assets/add_new_asset.md [hunk 2]
```diff
@@ -98,10 +144,10 @@ Process adding new tokens may look complicated at first glance, but once you com
 
 ### Easy way for Git user
 1. Fork the repo to your own github account
-2. Clone fork and crate new branch:
+2. Clone fork and create new branch:
 ```bash
 git clone git@github.com:YOUR_HANDLE/assets.git
-cd assets
+cd tokens
```
Comment (vikmeup) [inline assets/add_new_asset.md]: `cd assets` ?

File: assets/add_new_asset.md [hunk 3]
```diff
@@ -1,8 +1,22 @@
 # Trust Wallet Assets Info
+
+```diff
+- On November 8th 2019 we made big changes to the repository, please read summary and recomnadations
+```
+
+### What changed:
```
Comment (vikmeup) [inline assets/add_new_asset.md]: this is not needed here, keep it simple

you can add to release notes on github instead

Comment (kolya182) [reply assets/add_new_asset.md]: https://github.com/trustwallet/developer/pull/63/commits/53df752447fd2e3a8b755a2dff75b3b90fea70d4

File: assets/add_new_asset.md (hunk 2 above)
Comment (kolya182) [reply assets/add_new_asset.md]: https://github.com/trustwallet/developer/pull/63/commits/53df752447fd2e3a8b755a2dff75b3b90fea70d4

File: assets/add_new_asset.md (hunk 1 above)
Comment (kolya182) [reply assets/add_new_asset.md]: https://github.com/trustwallet/developer/pull/63/commits/53df752447fd2e3a8b755a2dff75b3b90fea70d4

**Pull request merged**